services/QuillLMS/app/controllers/concerns/growth_results_summary.rb

```diff
@@ -56,8 +56,8 @@ def growth_results_summary(pre_test_activity_id, post_test_activity_id, classroo
           total_possible_questions_count: total_possible_questions_count,
           total_pre_possible_questions_count: total_pre_possible_questions_count,
           total_maintained_skill_group_proficiency_count: total_maintained_skill_group_proficiency_count,
-          correct_question_text: "#{total_correct_questions_count} of #{total_possible_questions_count} questions correct",
-          correct_skill_groups_text: "#{total_correct_skill_groups_count} of #{skill_groups.count} skill groups"
+          correct_question_text: "#{total_correct_questions_count} of #{total_possible_questions_count} Questions Correct",
+          correct_skill_groups_text: "#{total_correct_skill_groups_count} of #{skill_groups.count} Skills"
```

Comment:
Did you confirm that we want to title-case these instead of sentence-casing them now? I always double check because generally we sentence-case things around the site, but a lot of the time the spec doesn't follow that rule so the incorrect casing makes it into the designs. I think if we're actually moving to title-case here, there are other places in these reports we'd want to update to be consistent.